plugin/pkg/dnsutil/ttl.go

```diff
@@ -47,7 +47,6 @@ func MinimalTTL(m *dns.Msg, mt response.Type) time.Duration {
 const (
 	// MinimalDefaultTTL is the absolute lowest TTL we use in CoreDNS.
 	MinimalDefaultTTL = 5 * time.Second
-	// MaximumDefaulTTL is the maximum TTL was use on RRsets in CoreDNS.
-	// TODO: rename as MaximumDefaultTTL
-	MaximumDefaulTTL = 1 * time.Hour
+	// MaximumDefaultTTL is the maximum TTL was use on RRsets in CoreDNS.
+	MaximumDefaultTTL = 1 * time.Hour
```

Comment:
This should be done as a deprecation.

```suggestion
	// MaximumDefaulTTL is the maximum TTL was use on RRsets in CoreDNS.
	// Deprecated: Use MaximumDefaultTTL.
	MaximumDefaulTTL = 1 * time.Hour
	// MaximumDefaultTTL is the maximum TTL was use on RRsets in CoreDNS.
	MaximumDefaultTTL = 1 * time.Hour
```